src/javascript/ast-value.ts

```diff
@@ -196,4 +198,21 @@ export function getIdentifierName(node: estree.Node): string|undefined {
   }
 }
 
+/**
+ * Formats the given identifier name under a namespace, if one is mentioned in
+ * the commentedNode's comment. Otherwise, name is returned.
+ */
+export function namespaceIdentifierName(
+    name: string, commentedNode: estree.Node): string {
```

Comment:
Can this take the parsed jsdoc, since presumably the caller has to access some tags and/or description anyway?